scripts/karma-rollup-plugin.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
```

Comment:
- is this a fork of something? 
- if so, do we need to retain a copyright notice? license?
- is it suitable to extract into its own module at a later time?

Reply:
also, please add a comment that explains why we need this and why the existing modules don't work for us

Reply:
Addressed in af057da9